storey/aggregations.py

```diff
@@ -317,6 +323,19 @@ async def get_or_save_schema(self):
 
         should_update = True
         if self._schema:
+            if self._use_windows_from_schema:
+                for aggr in self._aggregates:
+                    schema_aggr = self._schema[aggr.name]
+                    window_type = schema_aggr['window_type']
+                    window_secs = str(int(aggr.windows.max_window_millis / 1000)) + 's'
```

Comment:
I would actually enable `SlidingWindows` and `FixedWindows` to accept millis as period and windows in addition to strings. that way you won't need to do all these conversion just to get it back to millis in the end